deepcell/training.py

```diff
@@ -274,7 +278,7 @@ def train_model_conv(model,
         todays_date = datetime.datetime.now().strftime('%Y-%m-%d')
         data_name = os.path.splitext(os.path.basename(dataset))[0]
         model_name = '{}_{}_{}'.format(todays_date, data_name, expt)
-    model_path = os.path.join(model_dir, '{}.h5'.format(model_name))
+    model_path = os.path.join(model_dir, model_name)
```

Comment:
you can also format the model name with the epoch to prevent overwriting model files.